src/main/java/me/karakelley/tictactoe/Parser.java

```diff
@@ -0,0 +1,43 @@
+package me.karakelley.tictactoe;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.ArrayList;
+
+public class Parser {
+  private Reader filename;
+  private JSONParser jsonParser;
+  JSONObject jsonObject;
+
+  public Parser(FileReader filename, JSONParser jsonParser) throws IOException, ParseException {
+    this.filename = filename;
+    this.jsonParser = jsonParser;
+    this.jsonObject = convertFileToJsonObject();
+  }
+
+  public String firstMarker() {
+    return markers().get(1);
+  }
+
+  public String secondMarker() {
+    return markers().get(0);
```

Comment:
It's surprising to me that `firstMarker` returns the second element and `secondMarker` returns the first. One thing to keep in mind is this: "What information are people going to be interested in seeing when they're new to the codebase and they want to know how something works." I think it is really easy to over-refactor and obscure this knowledge.

My questions when looking at this class would be this: "I see a method named `firstMarker`. Where does that value come from in the JSON?" To answer that question in the current implementation I need go through this mental process...

"Ok, there's a method named markers which is returning a list of `String`s, but it looks like it's in the opposite order, and now `markers()` seems to be looking at a property called `"markers"` and whatever that value is it's calling `values()` on it." Nothing about the current implementation tells me about the `"playerOne"` or `"playerTwo"` keys, *and* it forces me to understand the JSON Simple library enough to understand what `values()` is doing.

To answer that question more quickly I would have implemented this like this...

```
pubic String firstMaker() {
  return json.get("markers").get("playerOne");
}

pubic String secondMarker() {
  return json.get("markers").get("playerTwo");
}
```

Now, all that being said, I think the lines are drawn kind of weird between `GameConfiguration` and `Parser`. I would not expect a class named `Parser` to know anything about the game of Tic Tac Toe.